Formula/a/afl-plus-plus.rb

```diff
@@ -0,0 +1,218 @@
+class AflPlusPlus < Formula
+  desc "American fuzzy lop plus plus: Security-oriented fuzzer, community-maintained"
+  homepage "https://github.com/AFLplusplus/AFLplusplus"
+  url "https://github.com/AFLplusplus/AFLplusplus.git",
+      tag:      "v4.10c",
+      revision: "775861ea94d00672c9e868db329073afd699b994"
+  version "4.10c"
+  license "Apache-2.0"
+  head "https://github.com/AFLplusplus/AFLplusplus.git",
+       branch: "dev"
+
+  depends_on "llvm"
+  depends_on "python@3.12"
+
+  conflicts_with "afl-fuzz", because: "they are different versions of the same software"
+
+  # Fix incompatibility with BSD coreutils.  Also, we don't want to
+  # alter the testing environment, and test_shm fails on macOS by
+  # default and hinders installation of working stuff so just remove
+  # that check entirely.
+  patch :DATA
```

Comment:
This patch should be submitted, reviewed and (at least) accepted by upstream before we ship it.